handlers/bounty.go

```diff
@@ -53,16 +53,21 @@ func (h *bountyHandler) GetBountyById(w http.ResponseWriter, r *http.Request) {
 	bountyId := chi.URLParam(r, "bountyId")
 	if bountyId == "" {
 		w.WriteHeader(http.StatusNotFound)
+		return
 	}
 	bounties, err := h.db.GetBountyById(bountyId)
 	if err != nil {
 		w.WriteHeader(http.StatusBadRequest)
 		fmt.Println("[bounty] Error", err)
-	} else {
-		var bountyResponse []db.BountyResponse = h.GenerateBountyResponse(bounties)
-		w.WriteHeader(http.StatusOK)
-		json.NewEncoder(w).Encode(bountyResponse)
+		return
 	}
+	if len(bounties) == 0 {
+		w.WriteHeader(http.StatusBadRequest)
```

Comment:
@elraphty, I will fix it shortly

Reply:
"Hi @elraphty, I checked and currently the logic is not correct because the unit test passes with the wrong ID. Please check it.

![image](https://github.com/stakwork/sphinx-tribes/assets/100023456/fb316d85-1af8-4978-803e-8f55f2b29bf9)

Reply:
@elraphty Fixed